repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -7,9 +7,10 @@
         "8": ["9.2"]
     },
     "saphana": {
-        "7.9": ["8.2", "8.6"],
-        "7": ["8.2", "8.6"],
+        "7.9": ["8.8", "8.6"],
```

Comment:
please sort the numbers from low to high

Reply:
the order is correct, since the latest in the list is the default upgrade path.

Reply:
discussed, 8.6 (and 9.0) is expected to keep as default for saphana